src/graphics/program-lib/chunks/cubeMapRotate.frag

```diff
@@ -0,0 +1,19 @@
+#ifdef CUBEMAPROT
+uniform mat3 cubeMapRotationMatrix;
+#endif
+
+vec3 cubeMapRotate0(vec3 refDir) {
```

Comment:
should these more descriptive suffix than 0 and 1?

Reply:
good idea - I will add NegX and PosX

Reply:
also done - thanks